README.md

```diff
@@ -2,6 +2,14 @@
 
 # <a href="https://gandi.net/"><img align="right" src="https://upload.wikimedia.org/wikipedia/en/c/c7/GandiSAS.svg" alt="awesome-gandi" title="awesome-gandi" style="width: 250px; height: 54px;" width="250" height="54"/></a> Awesome Gandi
 
+    A curated list of awesome Gandi tools, tutorials, and other resources.
+
+#### Contributing
+
+Contributions are welcome! To add a link or resource to the list, please [create an Issue](https://github.com/nlewo/awesome-gandi/issues), or fork this repository and submit a pull request ([click here to edit this file from github](https://github.com/nlewo/awesome-gandi/edit/main/README.md)).
```

Comment:
It would be nice to mention the `contributing.md` file.

Reply:
Indeed, done!